js/script.js

```diff
@@ -0,0 +1,64 @@
+let bookList = [];
+const formBook = document.querySelector('#book-form');
+const form = document.querySelectorAll('.form-input');
+const pageBookList = document.querySelector('.book-list');
+
+function updateStorage() {
+  const stringfiedBL = JSON.stringify(bookList);
+  localStorage.setItem('booklist', stringfiedBL);
+}
+
+function insertContent() {
+  bookList.forEach((book) => {
+    pageBookList.innerHTML += `
+         <article class="book">
+         <h4 class="book-name">${book.title}</h4>
+         <p class="book-writer">${book.author}</p>
+         <button class="remove-btn" id=${book.id} type="button" >Remove</button>
+         <hr>
+     </article>
+         `;
+  });
+  const removeBtn = document.querySelectorAll('.remove-btn');
+  removeBtn.forEach((button) => {
+    button.addEventListener('click', () => {
+      const newBookList = bookList.filter((book) => book.id.toString() !== button.id);
+      bookList = newBookList;
+      pageBookList.innerHTML = '';
+      insertContent();
+      updateStorage();
+    });
+  });
+}
+
+function updateUserInterface() {
+  const retrievedData = localStorage.getItem('booklist');
+  bookList = JSON.parse(retrievedData);
+  insertContent();
+}
+
+if (!localStorage.getItem('booklist')) {
+  updateStorage();
+} else {
+  updateUserInterface();
+}
+
+// random id
+const randomId = () => Math.round(Math.random() * 10000); // random id function
+
+function onSubmition(e) {
+  e.preventDefault();
+  const title = form[0].value;
+  const author = form[1].value;
+  const input = {
+    id: randomId(),
+    title,
+    author,
+  };
+  bookList.push(input);
```

Comment:
[OPTIONAL]
- It would be better to use methods that doesn't mutate the array, e.g. [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat), [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter), [slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) instead of [push](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push), [pop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/pop), [splice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice) and so on. 
See the example below:
```js
books = books.concat({id, title, author});
```